apps/editor/src/wysiwyg/wwEditor.ts

```diff
@@ -171,7 +171,7 @@ export default class WysiwygEditor extends EditorBase {
             (ev as ClipboardEvent).clipboardData || (window as WindowWithClipboard).clipboardData;
           const items = clipboardData?.items;
 
-          if (items?.length === 1) {
+          if (items) {
```

Comment:
In this case, data pasted from ms words and ms excel are inserted as images. 
Can you check the condition again and fix it so that they only work for images?

Reply:
shouldn't the next line (`const imageBlob = pasteImageOnly(items);`) take care of that?

Reply:
Sorry for the late response. 
In the case of ms word, not only `text/html` and `text/plain` but also data in the image format are stored in the clipboard.
Therefore, `items?.length === 1` is added and this condition is not present, an issue arises in which ms word text is pasted as an image regardless of the next line(`const imageBlob = pasteImageOnly(items); `). However, this condition raises an another issue(#2053) you wrote.

Could you look at this problem in more detail and solve it?

Reply:
i analyzed it a bit further:

if you copy from word / excel, you get **4** Items in your clipboard:

- text/plain the plaintext representation of the table
- text/html the html representation (html tags)
- text/rtf the rtf representation
- image/png the png file for the table


if you copy an image from google you get **2** items:

- text/html the html code to embedd the image (<img src="...">)
- image/png the png file

Here is some test-code you can play with:
```
<body>
  <ul id="cb-data">

  </ul>

  <script>
    const b = document.querySelector('body');
    const ul = document.querySelector('#cb-data');
    b.addEventListener('paste', (ev) => {

      while (ul.firstChild) {
        ul.removeChild(ul.lastChild);
      }


      for (var i = 0; i < ev.clipboardData.items.length; i++) {
        const item = ev.clipboardData.items[i];
        const li = document.createElement('li');
        li.innerHTML = "... items[" + i + "].kind = " + item.kind + " ; type = " + item.type;
        if (item.kind === "string") {
          item.getAsString(data => {
            const liul = document.createElement('ul');
            const liulli = document.createElement('li');
            liulli.innerHTML = data;
            liul.appendChild(liulli);
            li.appendChild(liul);
          });
        }
        ul.appendChild(li)
      }
    });
  </script>
</body>
```

Reply:
I added a new code to detect ms office copy data